lib/ret/storage.ex

```diff
@@ -198,52 +252,105 @@ defmodule Ret.Storage do
         end
 
         :filelib.fold_files(
-          Path.join(storage_path, @expiring_file_path),
+          Path.join(storage_path, expiring_file_path()),
           "\\.blob$",
           true,
           process_blob,
           nil
         )
 
-        # Clean empty dirs
         # TODO figure out what to do about owned files -- that structure increase over time
-        for type <- [@expiring_file_path] do
+        for type <- [expiring_file_path()] do
           root_path = "#{storage_path}/#{type}"
+          clean_empty_dirs(%{root_path: root_path})
```

Comment:
I think this may be one example where the named params pattern just feels redundant.

Reply:
Certainly if the language itself accommodated the pattern, I would have no reason to argue with it.